docs/post-pivot-configuration.md

```diff
@@ -0,0 +1,71 @@
+# Cluster configuration
+
+- [IBU/I post pivot configuration](#ibi-post-pivot-configuration)
+  - [Overview](#overview)
+  - [Network configuration](#network-configuration)
+  - [Recertification flow](#recertification-flow)
+  - [User specifications](#user-specifications)
+
+## Overview
+
+Post pivot configuration is on charge for applying user configurations after rebooting in new stateroot.
+- Perform networking configuration
+- Recertification flow
+- User specifications:
+  - Run recert flow
+  - Apply new hostname
+  - Apply new ip
+  - Apply all user specific configurations that user provides as https://github.com/openshift-kni/lifecycle-agent/blob/main/ibu-imager/clusterinfo/clusterinfo.go#L8
```

Comment:
@omertuc please update this doc with the proper link once https://github.com/openshift-kni/lifecycle-agent/pull/223 is merged

Reply:
Isn't the new IP and hostname part of the configuration the user provides here?